src/metabase/driver/bigquery.clj

```diff
@@ -351,7 +351,7 @@
         results (if mbql?
                   (post-process-mbql dataset-id table-name results)
                   (update results :columns (partial map keyword)))]
-    (assoc results :annotate? true)))
+    (assoc results :annotate? mbql?)))
```

Comment:
What's going on here?

I.e. this might be a good place to include a comment ;)